### PR TITLE
Bump gtfs lib

### PR DESCRIPTION
**Pull request by Tristramg**
Relates to etalab/transport-site#224

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,12 +38,6 @@
                         <configuration>
                             <finalName>datatool-server</finalName>
                             <transformers>
-                                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
```
Comment (bonjourmauko) [inline pom.xml]: Why did you delete these lines ?

Comment (Tristramg) [reply pom.xml]: It wasn’t me. It was upstream, and it wasn’t deleted just moved down.

At some point we’ll have to think how to manage the rebasing with upstream, as pull-requets won’t do the work.
